### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Create `UIPFINT` release
https://issues.folio.org/browse/UIPFINT-3
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "eslint": "^5.5.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-no-only-tests": "^2.3.0",
+    "eslint-plugin-jest": "^23.0.4",
```
Comment (NikitaSedyx) [inline package.json]: it should be before `eslint-plugin-no-only-tests`

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Pre-Merge Checklist
+Before merging this PR, please go through the following list and take appropriate actions.
+- [ ] I've added appropriate record to the CHANGELOG.md
+- Does this PR meet or exceed the expected quality standards?
+  - [ ] Code coverage on new code is 80% or greater
+  - [ ] Duplications on new code is 3% or less
+  - [ ] There are no major code smells or security issues
+- Does this introduce breaking changes?
+  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
+  - [*] There are no breaking changes in this PR.
```
Comment (aliaksei-chumakou) [inline PULL_REQUEST_TEMPLATE.md]: please remove the asterisk

File: package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "chai": "^4.2.0",
     "eslint": "^5.5.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-jest": "^23.0.4",
     "eslint-plugin-no-only-tests": "^2.3.0",
```
Comment (aliaksei-chumakou) [inline package.json]: I think `    "eslint-plugin-react-hooks": "^2.1.1", ` should be included as well

File: package.json [hunk 4]
```diff
@@ -36,7 +36,9 @@
...
+    "eslint-plugin-react-hooks": "^2.1.1",
```
Comment (aliaksei-chumakou) [inline package.json]: this is also not required

Comment (AndreiShumski2) [reply package.json]: Yes, without `@folio/stripes-acq-components`

**Pull request merged**